controllers/newscontroller.js

```diff
@@ -126,7 +126,7 @@ const paginateFeed = asyncHandler(async (req, res, next) => {
   const title = user.news.title;
   const date = user.news.date;
 
-  results.data = feeds.toJSON().slice(startIndex, endIndex);
+  results.data = feeds.slice(startIndex, endIndex);
```

Comment:
If you've used pagination on DB level then you can directly pass the response instead of slicing here